src/test/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterTest.java

```diff
@@ -51,33 +58,166 @@ public void testAccept() throws Exception {
     @Test
     public void testMultiChangesOnOneFileOne() throws Exception {
         final String fileName = getPath("MultiChangesOnOneFilePatch.txt");
-        final SuppressionPatchFilter filter = createSuppressionPatchFilter(fileName);
-        final LocalizedMessage message = new LocalizedMessage(4, 1, null, "msg", null,
-                SeverityLevel.ERROR, null, getClass(), null);
-        final AuditEvent ev = new AuditEvent(this, "Update.java", message);
-        assertTrue(filter.accept(ev),
-                "Audit event should be rejected when there are no matching patch filters");
+        final FileInputStream ins = new FileInputStream(fileName);
+        final UnifiedDiff diff = UnifiedDiffReader.parseUnifiedDiff(ins);
+        assertEquals(1, diff.getFiles().size());
+        final UnifiedDiffFile file1 = diff.getFiles().get(0);
+        assertNull(file1.getDiffCommand());
+        assertEquals("Origin.java", file1.getFromFile());
+        assertEquals("Update.java", file1.getToFile());
+        assertEquals(2, file1.getPatch().getDeltas().size());
+        assertNull(diff.getTail());
     }
 
     @Test
-    public void testMultiChangedFilesOnOnePatch() throws Exception {
+    @Ignore("https://github.com/java-diff-utils/java-diff-utils/issues/83")
+    public void testMultiChangedFilesOnOnePatchOne() throws Exception {
+        final String fileName = getPath("problem_diff_issue51.diff");
+        final FileInputStream ins = new FileInputStream(fileName);
+        final UnifiedDiff diff = UnifiedDiffReader.parseUnifiedDiff(ins);
+        assertEquals(2, diff.getFiles().size());
+        final UnifiedDiffFile file1 = diff.getFiles().get(0);
+        assertEquals("diff -U0 old/f1 new/f1", file1.getDiffCommand());
+        // here is f1 not old/f1 and new/f1
+        assertEquals("f1", file1.getFromFile());
+        assertEquals("f1", file1.getToFile());
```

Comment:
Is this what you expect or what you actually get from the lib?
Please write what you expect here, since you ignore this test because of the issue.
Should be `old/f1` and `new/f2` as I understand.

Reply:
yes, should be old/f1 and new/f2

Reply:
Still not fixed. Please put `old/f1` and `new/f1` in assetEquals. Once issue-83 is fixed in the library, this test should successfully pass.

Reply:
done